cluster/src/main/java/com/hartwig/pipeline/tertiary/cuppa/Cuppa.java

```diff
@@ -30,7 +34,13 @@
 import com.hartwig.pipeline.tertiary.purple.PurpleOutput;
 import com.hartwig.pipeline.tools.Versions;
 
+import org.jetbrains.annotations.NotNull;
+
 public class Cuppa implements Stage<CuppaOutput, SomaticRunMetadata> {
+    public static final String CUP_REPORT_SUMMARY_PNG = ".cup.report.summary.png";
+    public static final String CUP_DATA_CSV = ".cup.data.csv";
```

Comment:
I'd rather see `CUPPA_...` to match the others.

Reply:
the constants are extracted and named for the files. CUP has slightly different meaning in that it stands for "cancer of unknown primary" while CUPPA is the tool.